django/db/backends/mysql/features.py

```diff
@@ -119,11 +119,11 @@ def django_test_skips(self):
                 "update.tests.AdvancedTests.test_update_ordered_by_m2m_annotation_desc",
             },
         }
-        if self.connection.mysql_is_mariadb and (
+        if self.connection.mysql_is_mariadb and self.connection.mysql_version < (
             10,
-            4,
-            3,
-        ) < self.connection.mysql_version < (10, 5, 2):
+            5,
+            2,
+        ):
```

Comment:
If you're interested I was able to get black to behave better by wrapping the version range comparison in parens:

```
        if self.connection.mysql_is_mariadb and (
            self.connection.mysql_version < (10, 5, 2)
        ):
```

Reply:
> Edit: Seeing as this is a "problem" in a few places I'm happy to submit a separate PR after you're done updating all new versions? 👍

No need, this is not a problem. We switched to `black` to avoid such changes and discussions about preferred format.